components/RequiredYesBadge.vue

```diff
@@ -0,0 +1,11 @@
+<script setup>
+import { VPBadge } from 'vitepress/theme';
+</script>
+
+<template>
+  <VPBadge
+    type="tip"
+    text="required"
+    style="background-color: var(--vp-c-yellow-soft)"
+  />
+</template>
```

Comment:
How it looks like:

<img width="821" alt="Capture d’écran 2024-06-18 à 14 52 38" src="https://github.com/iExecBlockchainComputing/documentation-tools/assets/4280765/865b3dc2-7ce7-4d57-8e04-a2c0791f98c3">